src/main/webapp/app/modules/provider/provider-home.scss

```diff
@@ -451,3 +451,26 @@
   outline: none;
   display: grid !important;
 }
+
+.map-view {
+  padding-right: 0.5rem !important;
+}
+
+.selected-record {
+  padding-left: 1.5rem !important;
+}
+
+.selected-record > div {
+  margin-left: 0 !important;
+  margin-right: 0 !important;
+}
+
+@media only screen and (max-width: 768px) {
```

Comment:
Can it be used in inside .provider @media?

Reply:
sure